scripts/core/program.yml

```diff
@@ -391,7 +391,7 @@ etors:
     - name: STATUS
       desc: "Program build status, return type $x_program_build_status_t"
     - name: OPTIONS
-      desc: "Program build options, return type char[]"
+      desc: "Options string specified by last build, compile or link operation performed on the program. Return type char[]"
```

Comment:
Should maybe be `char*` for the return type and mention it being null-terminated.

Reply:
CL also does the [] thing but I don't think it makes a whole load of sense. I guess for char[] stuff it's like you're returning char[pPropSizeRet]? but that breaks down for stuff like int32_t[] so yeah changed all instances of that to use `*`